### ISSUE TITLE
The log reported an error every hour: [error] stats: deleting unit: bucket not found

### ISSUE THREAD

**Issue by KiceHub** [AdguardTeam/AdGuardHome#4850]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, OpenWrt

### CPU architecture

AMD64

### Installation

Docker

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.107.10 v0.107.11

#### What did you do?
I installed AdGuardHome using docker: 
```
 docker run --name AGH \
     -v /opt/AGH/work:/opt/adguardhome/work \
     -v /opt/AGH/conf:/opt/adguardhome/conf \
     -p 3000:3000 \
     -d aadguard/adguardhome:latest
```

This is my config, it's default: 
```
bind_host: 0.0.0.0
bind_port: 3000
beta_bind_port: 0
users:
  - name: admin
    password: *
auth_attempts: 5
block_auth_min: 15
http_proxy: ""
language: ""
debug_pprof: false
web_session_ttl: 720
dns:
  bind_hosts:
    - 0.0.0.0
  port: 53
  statistics_interval: 1
  querylog_enabled: true
  querylog_file_enabled: true
  querylog_interval: 2160h
  querylog_size_memory: 1000
  anonymize_client_ip: false
  protection_enabled: true
  blocking_mode: default
  blocking_ipv4: ""
  blocking_ipv6: ""
  blocked_response_ttl: 10
  parental_block_host: family-block.dns.adguard.com
  safebrowsing_block_host: standard-block.dns.adguard.com
  ratelimit: 20
  ratelimit_whitelist: []
  refuse_any: true
  upstream_dns:
    - https://dns10.quad9.net/dns-query
  upstream_dns_file: ""
  bootstrap_dns:
    - 9.9.9.10
    - 149.112.112.10
    - 2620:fe::10
    - 2620:fe::fe:10
  all_servers: false
  fastest_addr: false
  fastest_timeout: 1s
  allowed_clients: []
  disallowed_clients: []
  blocked_hosts:
    - version.bind
    - id.server
    - hostname.bind
  trusted_proxies:
    - 127.0.0.0/8
    - ::1/128
  cache_size: 4194304
  cache_ttl_min: 0
  cache_ttl_max: 0
  cache_optimistic: false
  bogus_nxdomain: []
  aaaa_disabled: false
  enable_dnssec: false
  edns_client_subnet: false
  max_goroutines: 300
  handle_ddr: true
  ipset: []
  filtering_enabled: true
  filters_update_interval: 24
  parental_enabled: false
  safesearch_enabled: false
  safebrowsing_enabled: false
  safebrowsing_cache_size: 1048576
  safesearch_cache_size: 1048576
  parental_cache_size: 1048576
  cache_time: 30
  rewrites: []
  blocked_services: []
  upstream_timeout: 10s
  private_networks: []
  use_private_ptr_resolvers: true
  local_ptr_upstreams: []
tls:
  enabled: false
  server_name: ""
  force_https: false
  port_https: 443
  port_dns_over_tls: 853
  port_dns_over_quic: 853
  port_dnscrypt: 0
  dnscrypt_config_file: ""
  allow_unencrypted_doh: false
  strict_sni_check: false
  certificate_chain: ""
  private_key: ""
  certificate_path: ""
  private_key_path: ""
filters:
  - enabled: true
    url: https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt
    name: AdGuard DNS filter
    id: 1
  - enabled: false
    url: https://adaway.org/hosts.txt
    name: AdAway Default Blocklist
    id: 2
whitelist_filters: []
user_rules: []
dhcp:
  enabled: false
  interface_name: ""
  local_domain_name: lan
  dhcpv4:
    gateway_ip: ""
    subnet_mask: ""
    range_start: ""
    range_end: ""
    lease_duration: 86400
    icmp_timeout_msec: 1000
    options: []
  dhcpv6:
    range_start: ""
    lease_duration: 86400
    ra_slaac_only: false
    ra_allow_slaac: false
clients:
  runtime_sources:
    whois: true
    arp: true
    rdns: true
    dhcp: true
    hosts: true
  persistent: []
log_file: ""
log_max_backups: 0
log_max_size: 100
log_max_age: 3
log_compress: false
log_localtime: false
verbose: false
os:
  group: ""
  user: ""
  rlimit_nofile: 0
schema_version: 14
```

#### Actual result
The service was actually running normally, but the log reported an error every hour: 
```
2022/08/19 15:00:00.348204 [error] stats: deleting unit: bucket not found
2022/08/19 16:00:00.865477 [error] stats: deleting unit: bucket not found
``` 
This bug started from v0.107.10 and did not appear before v0.107.9. It may be that some changes in v0.107.10 caused this bug.

**Comment by ainar-g** [comment]
Thanks for the report!

@EugeneOne1, please investigate.  If this isn't an actual actionable error, perhaps it's better to write it as a warning to the debug logs.

**Comment by chr-lei** [comment]
Just to add a bit more info - I also noticed this after moving to .10 and it still occurs on .11 as well. It seems to only log this error if the statistics retention is set for multiple days. I ran with the setting at 24 hours for about 2 days and had no issues, but moving back to 7 days started logging this error hourly.

**Comment by EugeneOne1** [comment]
@KiceHub, @chr-lei, hello again.  These errors were accidentally reported at the error level despite the fact that those are actually non-critical.  We've tried to fix it in the latest edge build (970b6cf6987cfadd108ae0db2cfa6747e345b9c1), could you please install it and check if the errors disappeared?
